src/features/organizations/rpc/getOrganizations.ts

```diff
@@ -1,52 +0,0 @@
-import { generateTreeData } from '../utils/generateTreeData';
-import IApiClient from 'core/api/client/IApiClient';
-import { makeRPCDef } from 'core/rpc/types';
-import { z } from 'zod';
-import {
-  ZetkinFile,
-  ZetkinMembership,
-  ZetkinOrganization,
-} from 'utils/types/zetkin';
-
-export interface TreeItemData {
-  avatar_file: ZetkinFile | null;
-  country: string | null;
-  email: string | null;
-  id: number;
-  is_active: boolean;
-  is_open: boolean;
-  is_public: boolean;
-  lang: string | null;
-  parent: { id: number; title: string } | null;
-  phone: string | null;
-  slug: string | null;
-  title: string;
-  children: TreeItemData[] | [];
```

Comment:
The `TreeItemData` type exists to be able to render a tree of organizations. That render logic only requires the ID, the title and the children. Since this type has no other purpose, all the other attributes are redundant. So I removed them, in order to make tests and all other code more concise.